turnerlabs/yt_push
# DRACO-3594 Upload videos in a background queue

Depends on pull request #4 

To test:

* Create a YouTube media type as described in the previous PRs
* Upload a video to the YouTube media type. After saving, a message will show that the video is queued to be uploaded.
* Run cron, or `drush queue-run yt_push_upload`. The video should be uploaded to YouTube, and after it's done the YouTube URL should show up in the media item.
